src/olympia/files/helpers.py

```diff
@@ -340,6 +308,51 @@ def get_syntax(self, filename):
                 return short
         return 'plain'
 
+    def _verify_files(self, verification_list, raise_on_verify=False):
+        """Verifies that all files are properly extracted.
+
+        TODO: This should probably be extracted into a separate helper
+        once we can verify that it works as expected.
+        """
+        strip = len(self.dest)
+
+        files_to_verify = get_all_files(self.dest)
+
+        # normalize file names, strip /tmp/xxxx/
+        files = [
+            '/'.join(fname.strip('/').split('/')[2:])
```

Comment:
`verification_list` is the list of files that gets passed down from `extract_xpi` which is using `tempfile.mkdtemp` as a target for it's initial extraction before files get copied to `settings.TEMP_PATH` (AWS EFS). Thus we can relatively safely assume that we have something like `/tmp/.../` here. Could be different on Windows but not sure here.